### ISSUE TITLE
CVE-2020-28483 github.com/gin-gonic/gin

### ISSUE THREAD

**Issue by riendeau** [OSSIndex/vulns#155]
**Advisory details**
```
  URL: https://nvd.nist.gov/vuln/detail/CVE-2020-28483
  format: golang
  namespace: <maven groupid, npm scope, etc.>
  name: github.com/gin-gonic/gin
  versions: [,1.7.0)
```

**More information**
https://ossindex.sonatype.org/component/pkg:golang/github.com/gin-gonic/gin@1.5.0 currently shows no vulnerabilities.

Fixed in 1.7.0 per https://snyk.io/vuln/SNYK-GOLANG-GITHUBCOMGINGONICGIN-1041736

**Comment by riendeau** [comment]
Fixed in 1.7.0 per https://snyk.io/vuln/SNYK-GOLANG-GITHUBCOMGINGONICGIN-1041736

**Comment by riendeau** [comment]
@ken-duck Any additional information needed on this? I still don't see this vulnerability reflected @ https://ossindex.sonatype.org/component/pkg:golang/github.com/gin-gonic/gin@1.5.0

**Comment by ultramaxim** [comment]
hello, why you still don't fix it? it seem to me, that snyk more relevant that your vuln db

**Comment by ken-duck** [comment]
This past year among other changes we upgraded the OSS Index vulnerability database. The new database has significantly more vulnerabilities, is much more actively maintained, and has more in depth research on many of the issues. The issue reported here was resolved  in the new database and as such we are closing the issue.

For the record, we are migrating to a new email-based reporting system in order to better mesh with our internal processes, which will allow us to be more reactive to our users.
